crates/orderbook-commons/Cargo.toml

```diff
@@ -11,3 +11,5 @@ secp256k1 = { version = "0.24.3", features = ["serde"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
 sha2 = { version = "0.10", default-features = false }
+time = "0.3.20"
+uuid = { version = "1.3.0", features = ["v4", "fast-rng", "macro-diagnostics", "serde"] }
```

Comment:
do we need all features?

Reply:
Sorry, copy paster, I'll remove the ones we don't need